src/BaGetter/appsettings.json

```diff
@@ -53,5 +53,10 @@
 
   "HealthCheck": {
     "Path" : "/health"
+  },
+
+  "Statistics": {
+    "EnableStatisticsPage": false,
```

Comment:
Why is it turned off by default? I'd prefer `true` defaults to show the new page. Services are debatable as that might be an attack vector, but then again BaGetter is mostly used internally so might not be important either.

Reply:
This makes actual sense. NuGet.org is the same.